packages/nimble-components/src/utilities/tests/fixture.ts

```diff
@@ -31,12 +31,6 @@ export interface FixtureOptions {
      */
     document?: Document;
 
```

Comment:
There's a comment at the top of the file tracking changes we've made versus the original FAST source. Who knows if we'll ever adopt their version (the linked issue is marked closed in the FAST Element 2.0 branch) but it couldn't hurt to add another comment mentioning this new diff.